components/brave_wallet_ui/components/desktop/hardware-wallet-connect/accounts_list.tsx

```diff
@@ -168,46 +141,31 @@ export const HardwareWalletAccountsList = ({
     onLoadMore()
   }
 
-  const isPreAddedAccount = React.useCallback(
-    (account: BraveWallet.HardwareWalletAccount) => {
-      return preAddedHardwareWalletAccounts.some(
-        (e) => e.address === account.address
-      )
-    },
-    [preAddedHardwareWalletAccounts]
-  )
-
   const onSelectNetwork = React.useCallback(
     (n: BraveWallet.NetworkInfo): void => {
       setSelectedNetworkId(networkEntityAdapter.selectId(n))
-      if (coin === BraveWallet.CoinType.FIL) {
-        onChangeFilecoinNetwork(n.chainId as FilecoinNetwork)
-      }
+      AllHardwareImportSchemes.map((scheme) => {
```

Comment:
Is there a need to map the schemes to a new array?
```suggestion
      AllHardwareImportSchemes.forEach((scheme) => {
```

Reply:
Also, are we looking for the last scheme? Or can the first scheme that is found be used?

```suggestion
      const schemeForNetwork = AllHardwareImportSchemes.find((scheme) => {
      
      ...
      ...
      
      if (schemeForNetwork) {
        setHardwareImportScheme(schemeForNetwork.derivationScheme)
      }
      
```